.github/workflows/validate.yml

```diff
@@ -1,29 +0,0 @@
-name: Validate
-
-on:
-  push:
-    branches:
-      - main
-  pull_request: {}
-
-jobs:
-  validate:
-    runs-on: ubuntu-latest
-    env:
-      # ruby/setup-ruby@v1 does not offer a way to change the cached gems path.
-      # See https://github.com/ruby/setup-ruby/issues/291
```

Comment:
Please can we preserve this explanatory comment. (Even though the issue has been closed as won't fix, having the reference is useful to explain what's going on here)